chapter2.adoc

```diff
@@ -44,41 +44,27 @@ When `mdbgen` is set to 0, the <<dbops, debug operations>> are disallowed and th
 ==== Supervisor Domain Debug Control
 The Smsdedbg extension cite:[smmtt] introduces `sdedbgalw` field (bit 7) in CSR <<Sdseccsr,msdcfg>>, to control the debuggability of supervisor domains. When `mdbgen`=0, the `sdedbgalw` field determines both the debug allowed privilege modes and the <<dbgaccpriv, debug access privilege>>, as illustrated in <<dbgpriv>>. 
 
+[[dbgaccpriv]]
+==== Debug Access Privilege
+The *debug access privilege* is defined as the privilege level granted to the external debugger when performing state accesses via the hart, such as Abstract Commands and Program Buffer execution. Memory and register accesses from Debug Mode carry *debug access privilege* instead of always with M-mode. The privilege is determined by <<mdbgctl, `mdbgen`>> and <<submdbgctl, `sdedbgalw`>>, as listed in <<dbgpriv>>. The `dmprv` field in `dcsr` modifies the memory load/store privilege, while leaving the register access privilege unaffected. When `dmprv`=1, the memory load and store uses the privilege level represented in `prv` and `v` fields. Any attempt by the debugger to access state inaccessible to *debug access privilege* will return an error and set `abstractcs.cmderr` to 3.
+
+[NOTE]
+When `dmprv`=1, the debugger memory access privilege matches the hart's privilege at Debug Mode entry. With `dmprv`, the debugger may modify the memory access privilege to VS/VU/U in Debug Mode when `mdbgen`=0 and `sdedbgalw`=1. However, the `dmprv` is still a debug feature, and it may be proposed to RISC-V Debug specification cite:[dbgspec] in the future to facilitate specific debug use cases. 
+
 [[dbgpriv]]
 [options="header"]
 [cols="15%,15%,35%,35%"]
 .External Debug Configuration and Privilege
 |============================================
-| mdbgen | sdedbgalw | Debug allowed privilege modes | Debug access privilege 
+| mdbgen | sdedbgalw | Debug allowed privilege modes | Debug access privilege (`dmprv`=0) 
 | 1      | Don't care      | All                      | M-mode
 | 0      | 1      | All except M                      | S-mode
 | 0      | 0      | None                              | N/A
 |============================================
 
-[[dbgaccpriv]]
-==== Debug Access Privilege
-
-The *debug access privilege* is defined as the privilege level granted to the external debugger when performing state accesses via the hart, such as Abstract Commands and Program Buffer execution.  Any attempt by the debugger to access state inaccessible to *debug access privilege* will return an error.
-
-[[prvvacc]]
-===== Configuring External Debugger Access Privileges
+==== Using EBREAK to Enter Debug Mode
```

Comment:
Why delete?

Reply:
Because in new proposal, the debug access privilege is not programmable but statically determined by `mdbgen` and `sdedbgalw` ( when `mdprv`=0) . The maximum resume privilege in `prv`/`v` is moved to CSR section below.

Reply:
Feels still useful to keep debug access privilege